one.mixin.messenger.yml

```diff
@@ -0,0 +1,45 @@
+id: one.mixin.messenger
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: mixin_desktop
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: mixin_desktop
+    buildsystem: simple
+    only-arches:
+    - x86_64
+    build-commands:
+    - mkdir -p FlutterApp
+    - tar -xf mixin_desktop_linux_amd64_portable.tar.gz -C FlutterApp
+    - cp -r FlutterApp /app/
+    - chmod +x /app/FlutterApp/mixin_desktop
+    - mkdir -p /app/bin
+    - ln -s /app/FlutterApp/mixin_desktop /app/bin/mixin_desktop
+    - install -Dm644 one.mixin.messenger.metainfo.xml -t /app/share/metainfo/
+    - install -Dm644 mixin_desktop.png /app/share/icons/hicolor/256x256/apps/one.mixin.messenger.png
```

Comment:
256 pixel icon ought to be enough. Since you don't (wont to) ship them upstream there is no need to have a collection here.

Reply:
done, 128x128 icon removed